src/components/bookings/Bookings.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from "react";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "../../data/fakeBookings.json";
+import Loading from "./Loading.js";
+import AddBookingModal from "./AddBookingModal.js";
+
+const Bookings = () => {
+  const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const [sorter, setSorter] = useState(false);
```

Comment:
What does `sorter` represent here? It looks like you're using it to change whether or not you're sorting ascending or descending.